code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -532,7 +532,7 @@
 			anchored = FALSE
 			playsound(loc, 'sound/machines/boltsup.ogg', 30, FALSE, 3)
 			return
-		to_chat(user, "<span class='warning'>You swipe your ID over the scanner, and with a steely snap bolts slide out from underneath the cart, securely anchoring it on the floor.</span>")
+		to_chat(user, "<span class='warning'>to_chat(user, "<span class='warning'>You swipe your ID over the scanner, and with a steely snap, bolts slide out from underneath the cart securely anchoring the cart to the floor.</span>")</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>You swipe your ID over the scanner, and with a steely snap, bolts slide out from underneath the cart securely anchoring the cart to the floor.</span>")
```

Reply:
Yeah, I did a dumb and decided to copy and paste - Fixed